web3js-ext/src/web3/klaytn_tx.ts

```diff
@@ -116,9 +123,10 @@ export class KlaytnTx extends LegacyTransaction {
     this._klaytnType = toNumber(txData.type) as number;
     this.from = txData.from;
     this.chainId = txData.chainId;
+    // @ts-ignore
+    this.key = txData.key;
 
-    // A readonly CoreKlaytnTx object
-    this.klaytnTxData = KlaytnTxFactory.fromObject({
+    let initTxData = {
```

Comment:
Name it klaytnTxObject

Reply:
Thanks https://github.com/klaytn/web3klaytn/pull/507/commits/854b25be74d13e7dd6d7b301849433e61e05e86b